doc/user_guide/ravenPostProcess.tex

```diff
@@ -65,7 +65,7 @@ \subsection{Build RAVEN Input: \xmlNode{PostProcess}}
      collect the outputs in the two \textit{DataObjects}. As it can be seen, the \xmlNode{Sampler} is inputted
      to communicate to the \textit{Step} that the driven code needs to be perturbed through the Monte Carlo sampling
      strategy.
-  \item \xmlNode{PostProcess} named ``statAnalysis\_1'', is used to compute all the statistical moments and FOMs
+  \item \xmlNode{PostProcess} named ``statAnalysis\_1'', is used to compute all the statistical moments and FOMs 
```

Comment:
trailing whitespace; you may be able to set your editor up to automatically remove this.

Reply:
Deleted